components/freertos/port/esp8266/port.c

```diff
@@ -237,17 +242,17 @@ void IRAM_ATTR vPortExitCritical(void)
                 }
             }
         } else {
-            ets_printf("E:C:%d\n", uxCriticalNesting);
+            ets_printf("E:C:%lu\n", uxCriticalNesting);
```

Comment:
Be aware that the `printf` function in the rom does not seem to support the 'l' prefix and this print would not work as expected. Casting `uxCriticalNesting` to unsigned and using "%u" would work for both the rom version and the newlib version of `printf`.